content/docs/command-reference/add.md

```diff
@@ -141,22 +145,16 @@ $ tree
 Let's check the `data.xml.dvc` file inside:
 
 ```yaml
-md5: aae37d74224b05178153acd94e15956b
 outs:
   - cache: true
```

Comment:
```suggestion
```

this one is no longer present by default.

Reply:
Thanks! Out of scope though, being removed in https://github.com/iterative/dvc.org/pull/1411/files#diff-1a0ca14da9037fa6403dcd8a33b4f16dL144 🙂

Reply:
Lots of overlapping docs PRs rn 😭